src/_pytest/compat.py

```diff
@@ -163,6 +163,7 @@ def getfuncargnames(
         and not isinstance(
             inspect.getattr_static(cls, name, default=None), staticmethod
         )
+        and not hasattr(function, "__self__")
```

Comment:
Can you explain this change?

Reply:
Parameterized tests are bound to an instance of the test class. And in their signature there is no longer a parameter for an instance of this class ('self').
A check on `bound` is needed so as not to delete the first fixture or parameter that is passed through arguments.